R/run-log.R

```diff
@@ -46,13 +46,20 @@ run_log <- function(.base_dir, .recurse = TRUE) {
 #' @importFrom purrr map_lgl map compact
 #' @importFrom fs dir_ls
 #' @keywords internal
-find_models <- function(.base_dir, .recurse) {
+find_models <- function(.base_dir, .recurse , .filter = 0:0) {
 
   # get yaml files
   yaml_files <- dir_ls(.base_dir, recurse = .recurse)
+  yaml_files <- subset(yaml_files, !(yaml_files %>% basename() %>% stringr::str_remove(".yaml") %in% {{.filter}}))
```

Comment:
Hmm, but then you're feeding the larger set through `basename` and `str_remove`.  I don't expect that'd be any better, though I haven't timed anything or otherwise missing something.  I think it is okay as you have it , though.